### PR TITLE
Adding siamese_network showcase

### PR DESCRIPTION
**Pull request by gaurav272333**

### CODE REVIEW

File: apphub/one_shot_learning/siamese_network.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+import numpy as np
+import tensorflow as tf
+from tensorflow.python.keras import Model, layers, Sequential
+from tensorflow.python.keras.initializers import RandomNormal
+from tensorflow.python.keras.regularizers import l2
+from tensorflow.keras.optimizers import Adam
+
+import fastestimator as fe
+from fastestimator.op import TensorOp
+from fastestimator.trace import Trace, LRController, Accuracy, ModelSaver, EarlyStopping
```
Comment (TortoiseHam) [inline apphub/one_shot_learning/siamese_network.py]: unused import

File: apphub/one_shot_learning/siamese_network.py [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+from fastestimator.schedule import LRSchedule
+from fastestimator.op.tensorop import ModelOp, BinaryCrossentropy, Minmax, Augmentation2D
+
+from fastestimator.dataset.omniglot import load_data, load_eval_data, get_batch, one_shot_trial
+
+
+class ConditionalAugmentation(TensorOp):
```
Comment (TortoiseHam) [inline apphub/one_shot_learning/siamese_network.py]: we should make a more general Op wrapper that can make any op conditional

Comment (TortoiseHam) [reply apphub/one_shot_learning/siamese_network.py]: possibly part of the base class?

**Comment by TortoiseHam** [comment]
Does it make sense for "one_shot_learning" to be it's own app hub folder? or should this be inside of image_classification or whatever it's actually doing?

File: apphub/one_shot_learning/siamese_network.py (hunk 1 above)
Comment (gaurav272333) [reply apphub/one_shot_learning/siamese_network.py]: Sure. I will remove that.

File: apphub/one_shot_learning/siamese_network.py (hunk 2 above)
Comment (vbvg2008) [reply apphub/one_shot_learning/siamese_network.py]: I implemented a general wrapper for that

**Pull request merged**